app/models/metric/capture.rb

```diff
@@ -31,210 +31,49 @@ def self.concurrent_requests(interval_name)
     requests
   end
 
-  def self.standard_capture_threshold(target)
-    target_key = target.class.base_model.to_s.underscore.to_sym
-    minutes_ago(::Settings.performance.capture_threshold[target_key] ||
-                ::Settings.performance.capture_threshold.default)
-  end
-
-  def self.alert_capture_threshold(target)
-    target_key = target.class.base_model.to_s.underscore.to_sym
-    minutes_ago(::Settings.performance.capture_threshold_with_alerts[target_key] ||
-                ::Settings.performance.capture_threshold_with_alerts.default)
-  end
-
+  # legacy messages on the queue
+  # went away in 4.6
```

Comment:
this `#went away in 4.6` still true? 
we need this for 4.6 coz 4.5 could have left behind queued item for this deprecated method.

Reply:
Well, the callers have all gone away.
I put this note in there to let people know that it is just around for legacy messages.

Are you thinking the scheduler should be paired down and still generate this message?
So all the intelligence will be in this process instead of the `Scheduler::Runner`?